lib/sass/calcite-web/type/_faces.scss

```diff
@@ -5,6 +5,7 @@
 //  ↳ type → _avenir-regular.md
 
 @mixin avenir-light() {
+  font-family: 'Avenir Next W05 Light','Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Comment:
unfortunately using font-family instead of font weight and font-style variants causes `b` and `strong` to not actually be bold anymore.